dozer/media/graphs.html

```diff
@@ -37,6 +37,12 @@ <h1><a href="%(home)s">Dozer</a>: Types</h1>
                 <input name="floor" value="%(floor)s"/>
                 <label>Filter</label>
                 <input name="filter" value="%(filter)s"/>
+                <label>SortBy</label>
+                <select name="sortby">
+                    <option value="monotonicity">monotonicity</option>
+                    <option value="-monotonicity">-monotonicity</option>
+                    <option value="none"></option>
+                </select>

```

Comment:
Use "none" as default

Reply:
The display is inconsistent with actual parameter. Pass the selection back to front end and set option attribute